action.yml

```diff
@@ -54,10 +54,10 @@ runs:
       working-directory: ${{ runner.temp }}
       env:
         GHLINT_VERSION: ${{ inputs.version }}
+        GH_TOKEN: ${{ github.token }}
```

Comment:
Seems required even though it's not used

<img width="840" alt="image" src="https://github.com/TWiStErRob/net.twisterrob.ghlint/assets/2906988/1f17fcb4-6be6-4580-bd97-a9fb5daf68bc">